pmd-core/src/test/java/net/sourceforge/pmd/lang/ast/AbstractNodeTest.java

```diff
@@ -228,24 +226,4 @@ public void testRemoveChildAtIndexOnNodeWithNoChildren(final int grandChildIndex
         assertEquals(0, grandChild.getNumChildren());
     }
 
-
-    @Test
-    public void testDeprecatedAttributeXPathQuery() throws JaxenException {
```

Comment:
I'll readd the test - although Jaxen / findChildNodesWithXPath is deprecated, that does not prevent anyone from using it... we ourselves use it in a couple of rules...